go/exocom/main_test.go

```diff
@@ -0,0 +1,128 @@
+package main_test
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"os/exec"
+	"strings"
+	"testing"
+
+	"github.com/DATA-DOG/godog"
+	"github.com/Originate/exocom/go/exocom/test_helpers"
+)
+
+// nolint gocyclo
+func FeatureContext(s *godog.Suite) {
+	var cmd *exec.Cmd
+	var cmdOutput string
+	var cmdStdout io.ReadCloser
+	var cmdStderr io.ReadCloser
```

Comment:
Sounds good. I'll extract out process helpers from exosphere into a new repo and can use that in both places. I'll handle that in a separate PR though